README.md

```diff
@@ -4,19 +4,33 @@
 
 [![npm](https://img.shields.io/npm/v/rdme)](https://npm.im/rdme) [![Build](https://github.com/readmeio/rdme/workflows/CI/badge.svg)](https://github.com/readmeio/rdme)
 
-`rdme` is the CLI wrapper for [ReadMe's RESTful API](https://docs.readme.com/reference/intro-to-the-readme-api). It allows you to upload and edit [OpenAPI](https://swagger.io/specification/) and [Swagger](https://swagger.io/specification/v2/) files associated with projects you create on [ReadMe](https://readme.com/). Additionally, you can sync documentation with your project, and manage project versions.
+`rdme` is the CLI and [GitHub Action](https://docs.github.com/actions) wrapper for [ReadMe's RESTful API](https://docs.readme.com/reference/intro-to-the-readme-api). It allows you to sync [OpenAPI](https://spec.openapis.org) and [Swagger](https://swagger.io/specification/v2/) files with projects you create on [ReadMe](https://readme.com/). Additionally, you can sync documentation with your project, and manage project versions.
 
 ## Configuration
 
-### Installation
+### Setup
+
+> These setup instructions are for CLI usage only. For usage in GitHub Actions, see [GitHub Actions](#github-actions) below.
+
+We recommend installing `rdme` in your project's `devDependencies` so you don't run into unexpected behavior with mismatching versions:
```

Comment:
Why does `devDependencies` prevent "unexpected behavior with mismatching versions"? (Is there a clearer reason why to recommend using `devDependencies`, like "because you don't need it in running on your production servers"?)

Reply:
Ah that's a great callout. The reasoning for `rdme` being installed in `devDeps` is separate from the reasoning for mismatching versions... I'll clarify!